targets/lenovo-x1/guivmExtraModules.nix

```diff
@@ -136,24 +136,28 @@
     # Early KMS needed for GNOME to work inside GuiVM
     boot.initrd.kernelModules = ["i915"];
 
-    microvm.qemu.extraArgs = [
-      # Lenovo X1 Lid button
-      "-device"
-      "button"
-      # Lenovo X1 battery
-      "-device"
-      "battery"
-      # Lenovo X1 AC adapter
-      "-device"
-      "acad"
-      # Connect sound device to hosts pulseaudio socket
-      "-audiodev"
-      "pa,id=pa1,server=unix:/run/pulse/native"
-    ];
+    microvm.qemu.extraArgs =
+      [
+        # Lenovo X1 Lid button
+        "-device"
+        "button"
+        # Lenovo X1 battery
+        "-device"
+        "battery"
+        # Lenovo X1 AC adapter
+        "-device"
+        "acad"
+        # Connect sound device to hosts pulseaudio socket
+        "-audiodev"
+        "pa,id=pa1,server=unix:/run/pulse/native"
+      ]
+      ++ lib.optionals configH.ghaf.hardware.fprint.enable configH.ghaf.hardware.fprint.qemuExtraArgs;
```

Comment:
More of a general comment and not scope of this pull request, but it might make sense to use [toGNUCommandLine](https://github.com/NixOS/nixpkgs/blob/0d28066770464d19d637f6e8e42e8688420b6ac6/lib/cli.nix#L31) for qemu extraArgs at some point given in how many places it is set. The thinking is that than one could make use of the module system and do overrides in one module for where default was set in a different one.